schemas/bigip-virtual-server_v0.1.4.json

```diff
@@ -182,7 +182,7 @@
       "type": "object",
```

Comment:
Question on this file: do we now require that the controller configs for the latest version require this schema file?  If so, it should be called out in the docs. The current docs list that v1.1.0 or higher require v0.1.3 of the schema.

Reply:
I believe from the comments on the v0.1.4 schema, the change for supporting multiple SSL profiles is backwards compatible.  The current change for supporting bigip style addresses is also backwards compatible. This was verified with the systests which currently use v0.1.3 of the schema.